### PR TITLE
feat: Add call for speaker section

### PR DESCRIPTION
**Pull request by anhanh11001**
Detail:
- Set up menu action to get to Speakers Call Section
- I haven't implemented the submitting proposal feature so the button is not working now as I don't want to make a large PR. I'll open a new issue after this get merged to implement that.
- Putting Speakers Call inside menu item is just temporary. I'll open a new issue later to set up the tab layout so that every sections in EventDetailsFragment are placed logically and user-friendly.

Fixes: #1631 

Screenshots for the change:
              
ezgif-2-23f1dc967d2d

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/OpenEventDatabase.kt [hunk 1]
```diff
@@ -41,7 +43,7 @@ import org.fossasia.openevent.general.ticket.TicketIdConverter
 
 @Database(entities = [Event::class, User::class, SocialLink::class, Ticket::class, Attendee::class,
     EventTopic::class, Order::class, CustomForm::class, Speaker::class, SpeakerWithEvent::class, Sponsor::class,
-    SponsorWithEvent::class, Session::class], version = 6)
+    SponsorWithEvent::class, Session::class, SpeakersCall::class], version = 6)
```
Comment (liveHarshit) [inline app/src/main/java/org/fossasia/openevent/general/OpenEventDatabase.kt]: I think json file for version = 6 is not committed in development?

File: app/src/main/java/org/fossasia/openevent/general/speakercall/SpeakersCallViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            mutableError.value = resource.getString(R.string.error_fetching_event_section_message,
+                resource.getString(R.string.speakers_call))
+            mutableProgress.value = false
+            return
+        }
+
+        compositeDisposable += eventService.getSpeakerCall(eventId)
+            .withDefaultSchedulers()
+            .doFinally {
+                mutableProgress.value = false
```
Comment (liveHarshit) [inline app/src/main/java/org/fossasia/openevent/general/speakercall/SpeakersCallViewModel.kt]: IMO it's better if you set default visibility of progress to GONE and set the mutableProgress value to true onSubscribe.

File: app/src/main/java/org/fossasia/openevent/general/OpenEventDatabase.kt (hunk 1 above)
Comment (anhanh11001) [reply app/src/main/java/org/fossasia/openevent/general/OpenEventDatabase.kt]: Yes, it isn't

**Comment by anhanh11001** [comment]
updated

**Pull request merged**